src/main/java/response/HttpResponse.java

```diff
@@ -23,12 +23,14 @@ public int responseCode() {
     return responseCode;
   }
 
-  public void setBody(byte[] body) {
+  public void setBody(byte[] body, String contentType) {
```

Comment:
I can see the `Content-Length` getting set here–nice job–_but_ I feel like the `contentType` is out of place here. Feel like it should be a separate setter altogether.